README.md

```diff
@@ -93,22 +86,11 @@ namespace switching. You can use the following aliases:
 alias kctx='kubectx'
 alias kns='kubens'
 ```
-#### Kubernetes Metrics Server
-
-To get some metrics from nodes or pods, you can use [Kubernetes Metrics
-Server](https://github.com/kubernetes-incubator/metrics-server). There are 
-the aliases
-
-```bash
-alias kt='kubectl top'
-alias ktn='kubectl top nodes'
-alias ktp='kubectl top pods'
-```
 
-### Trouble Shooting
+# Customizing Aliases
 
-#### Autocomplete
+The `aliases.yaml` is used to generate aliases. It can be modified and aliases
```

Comment:
A thought, since updating this plugin might over-write that file, should we recommend people create a different file?

Reply:
Oo yea. Most definitely.